torch/nn/parallel/scatter_gather.py

```diff
@@ -43,11 +59,17 @@ def scatter_map(obj):
     try:
         res = scatter_map(inputs)
     finally:
-        scatter_map = None
+        scatter_map = None  # type: ignore[assignment]
     return res
 
 
-def scatter_kwargs(inputs, kwargs, target_gpus, dim=0):
+# TODO More precise types here.
```

Comment:
FYI: https://github.com/Instagram/MonkeyType can help you find more precise types here (if you remove the Any annotations).